### ISSUE TITLE
osc branch a package into another project for scmsync package

### ISSUE THREAD

**Issue by mcepl** [openSUSE/osc#1381]
**Describe the bug**
With a regular (OSC managed) package I can do

```sh
 $ osc branch openSUSE:Factory par_text home:mcepl:neovim
```

to have the package available in another project (because it is required for dependency or it should be build in a different environment or for different platform).

With scmsync managed package I get an error:

```
tumbleweed-pkg~/b/neovim$ osc branch openSUSE:Factory par_text home:mcepl:neovim
BuildService API error: failed to branch: <status code="404">
  <summary>_link: no such file</summary>
  <details>404 _link: no such file</details>
</status>

tumbleweed-pkg~/b/neovim$ 
```

Any ideas?

**Versions**
- Operating system: Tumbleweed
- Packages: local build from the master branch (currently tag 1.3.0).

**To Reproduce**
1. see above

**Expected behavior**
somehow the package becomes present in the other package as well

**Comment by adrianschroeter** [comment]
for one we will definitive not supporting source merge mechanism of native OBS packages together with git managed sources.

I wonder why you even try to use it, because you may not know that a package is actually git managed?
In that case we should just report an error message pointing to an explanation how to work with gitea instead IMHO.

What we need to find is a tool/workflow how one can setup easily a test build env after ones has forked a package in any git server IMHO.

**Comment by hennevogel** [comment]
This is solved by openSUSE/open-build-service#14752 for packages you convert from now on and on build.o.o also for packages that have been converted previously.

What is not solved is that branching does not "transfer" the scmsync attribute 👉🏿 openSUSE/open-build-service#14803

**Comment by mcepl** [comment]
This bug is probably irrelevant now and it could be closed.
